examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -21,8 +21,13 @@
 #############################################################################
 # Load the behavioral data
 import numpy as np
-y, session = np.loadtxt(haxby_dataset.session_target[0]).astype("int").T
-conditions = np.recfromtxt(haxby_dataset.conditions_target[0])['f0']
+labels = np.recfromcsv(haxby_dataset.session_target[0], delimiter=" ")
+conditions = labels['labels']
+categories = np.unique(conditions)
+y = np.zeros_like(conditions)
+for c, category in enumerate(categories):
+    y[conditions == category] = c
```

Comment:
You're right: I was so focused on getting the y vector the way it was on the simple fetcher, I forgot I could just take the conditions. 

I just commited the simplifcations, except for `plot_haxby_mass_univariate`, that (at least the way it's coded now) requires both an unmasked conditions vector and a masked int-encoded one. Perhaps it's less problematic to keep it that way since it's supposed to be an advanced example. What do you think?